src/puyapy/awst_build/arc4_decorators.py

```diff
@@ -231,6 +231,8 @@ def visit_name_expr(self, o: mypy.nodes.NameExpr) -> object:
         elif self.arg_name == _CLIENT_DEFAULTS:
             if isinstance(o.node, mypy.nodes.Decorator):
                 return o.name  # assume abimethod
+            if o.fullname in (["builtins.True", "builtins.False"]):
```

Comment:
lint: I'm surprised ruff didn't flag this, but the () here is redundant (and in this case a bit confusing because it looks like a list within a tuple until you see the lack of comma).